index.php

```diff
@@ -4,4 +4,21 @@
  * User: mzikl
  * Date: 12.5.2018
  * Time: 13:47
- */
\ No newline at end of file
+ */
+
+$title = 'Uvodna stranka';
+
+require("php/subpages/layout/header.php");
+?>
+
+<div class="container">
+    <div class="row">
```

Comment:
Z akeho dovodu mas v indexe napevno registraciu bez loginu?

Reply:
Dal som ti to upravit lebo tam mas len zaregistrujte sa. Ale na stranke by si mal mat stale aj moznost sa prihlasit bez toho aby si klikal registrujte sa a potom prihlaste sa.